lib/Db/CollectiveShare.php

```diff
@@ -37,6 +38,14 @@ public function setEditable(bool $editable): void {
 		$this->editable = $editable;
 	}
 
+	public function getPassword(): string {
+		return $this->password;
+	}
+
+	public function setPassword(string $password): void {
+		$this->password = $password;
+	}
+
```

Comment:
Is there any difference between this and the `@method` comment used above for `getToken` and `setToken`?

Reply:
Yes,  `@method` injects getters and setters for database fields. For transient attributes in the class that don't represent a database field, we have to define them manually. I just learned about this recently myself, when merging the `CollectiveShareInfo` and `CollectiveShare` classes. See https://docs.nextcloud.com/server/latest/developer_manual/basics/storage/database.html#transient-attributes